lib/command.ts

```diff
@@ -602,7 +596,7 @@ export class CommandInstance {
       });
 
       Object.keys(parsed.argv).forEach(key => {
-        if (positionalKeys.indexOf(key) !== -1) {
+        if (~positionalKeys.indexOf(key)) {
```

Comment:
I think here you could just use `.includes` now?

Reply:
Sounds good to me.